### ISSUE TITLE
kubectl port-forward - invalid error message.

### ISSUE THREAD

**Issue by marekbiskup** [GoogleCloudPlatform/kubernetes#11309]
```
biskup@cathedral:~/go/src/github.com/GoogleCloudPlatform/kubernetes$ kubectl port-forward 
error: POD is required for exec
```

Should be "for port-forward"

**Comment by yujuhong** [comment]
Are you using the kubectl from gcloud?

```
$ ./cluster/kubectl.sh port-forward
error: POD_NAME is required for port-forward
see 'kubectl port-forward -h' for help.
```

```
$ which kubectl
/usr/local/google/home/abc/google-cloud-sdk/bin/kubectl
$ kubectl port-forward
error: POD is required for exec
see 'kubectl port-forward -h' for help.
```

**Comment by feihujiang** [comment]
this was addressed by #10913
